recipes/aoe2netwrapper/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This is already included in the PyPI tarball used to build the package, so it should not be included here. Please go ahead and remove.